### DISCUSSION TITLE
Any plan to support Eclipse Application Launch Configuration ?

### DISCUSSION THREAD

**Discussion by hengsin** [JaneWardSandy/eclipse-pde-partial-idea#65] [General]
Hi, is there any plan to add support for Eclipse Application Launch Configuration ? That will help a lot to ease the pain to migrate from Eclipse.

**Comment by JaneWardSandy** [comment]
I had migrate some config from `Eclipse Application Launch Configuration`, and didn't get another part that I don't use it.

<img width="804" alt="截屏2022-02-18 10 41 21" src="https://user-images.githubusercontent.com/15345355/154608132-779a200e-b79c-426e-818f-b675f0fbe426.png">
<img width="799" alt="截屏2022-02-18 10 43 51" src="https://user-images.githubusercontent.com/15345355/154608146-a4b91ef8-6b65-467e-9416-23d9e1ca4a3a.png">
<img width="799" alt="截屏2022-02-18 10 43 51" src="https://user-images.githubusercontent.com/15345355/154608157-e59f260e-df03-4902-a0fc-2ed4407d1f96.png">
<img width="803" alt="截屏2022-02-18 10 44 46" src="https://user-images.githubusercontent.com/15345355/154608167-c14d003b-0863-4957-ac52-6cc5e6596a1b.png">
<img width="1331" alt="截屏2022-02-18 10 46 55" src="https://user-images.githubusercontent.com/15345355/154608178-5e6498f6-04d6-4b37-b3bb-ed40a9eb10e9.png">

**Comment by JaneWardSandy** [comment]
`plugins` tab was not supported, now it only had an global `contents` tab to check and validate plugins dependencies.
<img width="744" alt="截屏2022-02-18 10 50 22" src="https://user-images.githubusercontent.com/15345355/154608587-ad3ee568-82a2-43f2-9bac-c06caeab5fc8.png">
<img width="798" alt="截屏2022-02-18 10 50 38" src="https://user-images.githubusercontent.com/15345355/154608598-78fd10fc-3dd6-4fd7-9e74-37ad93690fb7.png">
<img width="729" alt="截屏2022-02-18 10 50 55" src="https://user-images.githubusercontent.com/15345355/154608600-81d3570c-57fb-40bd-b6d0-49b8d608181e.png">

**Comment by hengsin** [comment]
Well, the global content tabs approach doesn't works well when you have > 1 Eclipse application in your project. In comparison, https://marketplace.visualstudio.com/items?itemName=yaozheng.vscode-pde works fine with support for Eclipse Target Platform definition file and Eclipse Application Launcher configuration file.

**Reply by JaneWardSandy** [reply]
That sound great, I will take the time to view the source code of the plug-in and learn to improve related functions.
